docs/references/react/use-organization-list.mdx

```diff
@@ -49,6 +51,19 @@ const JoinedOrganizationList = () => {
       >
         Load more
       </button>
+
+      <button
+        disabled={!isLoaded}
+        onClick={async () => {
+          await createOrganization({
+              name: 'randomOrg' + Date.now(),
+              slug: 'random-org' + Date.now(),
```

Comment:
We can omit slug here as it's optional and we are looking to phase it out as we are moving forward